src/components/table/Table.tsx

```diff
@@ -191,7 +190,7 @@ export const Table: FC<ITable> = forwardRef(
 
         firstRowRef.current.style.height = height + "px";
       }
-      debounce(() => triggerLoadMoreItems(), 160);
+      debounce(triggerLoadMoreItems, 160)();
```

Comment:
@yousseefben this change is only making the call to be delayed 160ms, it is not debouncing it. I've already addressed this on this PR https://github.com/caisy-io/league/pull/405/files

Reply:
@MoritzGruber I've just talked to Youssef about this one, and we think we can kill this PR in favor of the other one in my comment.